shared/checks/oval/system_boot_mode_is_uefi.xml

```diff
@@ -0,0 +1,24 @@
+<def-group>
+  <definition class="compliance" id="system_boot_mode_is_uefi" version="1">
+    <metadata>
+      <title>UEFI system boot mode check</title>
+      <affected family="unix">
+        <platform>multi_platform_all</platform>
+      </affected>
+      <description>Check if system boot mode is UEFI.</description>
+    </metadata>
+    <criteria operator="AND">
+      <criterion test_ref="test_efi_dir_existence" comment="check if /sys/firmware/efi folder exists" />
+    </criteria>
+  </definition>
+
+  <unix:file_test check="all" check_existence="at_least_one_exists" comment="Test if /sys/firmware/efi folder exists" id="test_efi_dir_existence" version="1">
+    <unix:object object_ref="object_file_sys_firmware_efi" />
+  </unix:file_test>
+
+  <unix:file_object comment="/sys/firmware/efi" id="object_file_sys_firmware_efi" version="1">
+    <unix:path>/sys/firmware/efi</unix:path>
+    <unix:filename xsi:nil="true" />
```

Comment:
Collapse lines 23-24 to:
```<ind:filepath>/sys/firmware/efi</ind:filepath>```

Reply:
/sys/firmware/efi is a directory and according to OVAL spec directory can't be specified as a filepath:
https://oval.mitre.org/language/version5.11/ovaldefinition/documentation/unix-definitions-schema.html#file_object
@redhatrises is suggested change still relevant?